e2e/provision/playbooks/roles/bootstrap/tasks/load-sctp-module.yml

```
@@ -0,0 +1,21 @@
+---
+# SPDX-license-identifier: Apache-2.0
+##############################################################################
+# Copyright (c) 2023 The Nephio Authors.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Apache License, Version 2.0
+# which accompanies this distribution, and is available at
+# http://www.apache.org/licenses/LICENSE-2.0
+##############################################################################
+
+- name: install sctp on {{ ansible_os_family }}
+  become: yes
+  ansible.builtin.package:
+    name: "kernel-modules-extra-{{ ansible_kernel }}"
```

I'm getting the following error

```console
TASK [bootstrap : install sctp on RedHat] **************************************
fatal: [fedora34]: FAILED! => {"changed": false, "failures": ["No package kernel-modules-extra-5.17.12-100.fc34.x86_64 available."], "msg": "Failed to install some of the specified packages", "rc": 1, "results": []}
```

Can you try to use the generic package `kernel-modules-extra`?
```suggestion
    name: kernel-modules-extra
```

---

strange, It works fine on my Fedora34 cloud image gcp VM. Its important that the package being pulled matches the kernel version installed.